lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -1095,7 +1095,7 @@ var _ = Describe("Test NetworkPolicy conversion (k8s <= 1.7, no policyTypes)", f
 								PodSelector: &metav1.LabelSelector{
 									MatchLabels: map[string]string{
 										"k":  "v",
-										"k2": "v2",
+										"k2": "v3",
```

Comment:
didn't really need to change this - but more hassle that it's worth to undo just this.

Reply:
ah okay, you can disregard previous comments on spurious `LabelSelector` changes